cmd/debos/debos.go

```diff
@@ -94,8 +95,10 @@ func main() {
 	}()
 
 	parser := flags.NewParser(&options, flags.Default)
-	args, err := parser.Parse()
+	opt := parser.FindOptionByLongName("backend")
+	opt.Choices = fakemachine.BackendNames()
```

Comment:
I wonder if  fakemachine vs. non-fakemachine should be folded into the backend choice hwere:
auto means a support fakemachine setup *or* the non-fakemachine setup; backend kvm should fail if there is no fakemachine support at all. while there could be a non-fakemachine backend which is our old --disable-fakemachine essentially?

alternatively this option should be "fakemachine backend" specifically so it's more clear